src/current/_includes/v24.3/cdc/msk-tutorial-crdb-setup.md

```diff
@@ -0,0 +1,41 @@
+1. (Optional) On the EC2 instance running CockroachDB, run the [Movr]({% link {{ page.version.version }}/movr.md %}) application workload to set up some data for your changefeed.
+
+    Create the schema for the workload:
+
+     {% include_cached copy-clipboard.html %}
+     ~~~shell
+     cockroach workload init movr
+     ~~~
+
+     Then run the workload:
+
+     {% include_cached copy-clipboard.html %}
+     ~~~shell
+     cockroach workload run movr --duration=1m
+     ~~~
+
+1. Start a SQL session. For details on the available flags, refer to the [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}) page.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --insecure
+    ~~~
+
+1. Set your organization name and [{{ site.data.products.enterprise }} license]({% link {{ page.version.version }}/enterprise-licensing.md %}) key:
```

Comment:
strongly request that:

- we do not replicate these instructions here, and instead point users to `{% link {{ page.version.version }}/licensing-faqs.md %}#set-a-license`
- we do not point at `({% link {{ page.version.version }}/enterprise-licensing.md %})` which is getting removed in the next ~2 weeks due to there being no reason for that page to exist post-November (since there is no such thing as "not Enterprise")

PS not quite rising to a blocker but please i'm begging you :-)